changelogs/unreleased/gh-5444-relax-uuid-validation.md

```diff
@@ -0,0 +1,3 @@
+## feature/core
+
+* Any 128-byte sequence is considered as a valid UUID value (gh-5444).
```

Comment:
```suggestion
* Any 128-byte sequence is considered as a valid UUID value to support all RFC 4122 UUID versions (gh-5444).
```

Reply:
I've mentioned RFC 9562 instead, since it is the latest one and its examples are already used in the test.